ui/src/app/pages/homepage/recent-resources.tsx

```diff
@@ -114,7 +114,8 @@ export const RecentResources = fp.flow(withCdrVersions())((props: Props) => {
 
     const getCdrVersionName = (r: WorkspaceResource) => {
       const {cdrVersionTiersResponse} = props;
-      return getCdrVersion(getWorkspace(r), cdrVersionTiersResponse).name;
+      const cdrVersion = getCdrVersion(getWorkspace(r), cdrVersionTiersResponse);
```

Comment:
I ran into an issue on local where the cdrVersion was not defined. It may just be my local setup, but it was annoying

Reply:
hm.  I'll make a note too look into this.

Reply:
thanks!